spec/models/politician_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
```

Comment:
@drod1000 confirmed this message is not applicable to our implementation due to Ruby version we use vs what Ruby version Hound is using. FYI, @njgheorghita @Dpalazzari